packages/flutter_tools/lib/src/features.dart

```diff
@@ -124,10 +124,6 @@ const Feature flutterBuildPluginAsAarFeature = Feature(
   name: 'Build plugins independently as AARs in app projects',
   configSetting: 'enable-build-plugin-as-aar',
   master: FeatureChannelSetting(
-    available: true,
```

Comment:
If you make this available: true. Then it can still be manually enabled with `flutter config --enable-build-plugin-as-aar`.